dpub-aria/index.html

```diff
@@ -21,16 +21,15 @@
         editors: [
           {
             name: "Matt Garrish",
-            url: "https://www.daisy.org",
             company: "DAISY Consortium",
-            companyURI: "https://www.daisy.org",
+            companyURI: "https://daisy.org",
```

Comment:
```suggestion
            companyURL: "https://daisy.org",
```

That key was incorrect.

Reply:
Thanks, yes, it looks like they've mapped it forward to the new name but I've changed the other uses that were in the metadata.